src/js/components/footer.js

```diff
@@ -0,0 +1,41 @@
+import { newText, newMultimedia, newLink, newContainer } from './helpers';
+
+const createlinks = (datalinks) =>{
+  const arraylinks = datalinks.map((link)=>{
+    const linksitems = newLink(link.href,link.text,['footer__data-links']);
+    const listlink = newContainer('li',[linksitems],['footer__items-list']);
+    return listlink;
+  });
+  return arraylinks;
+}
+
+const createicons = (dataicons) => {
+  const arrayicons = dataicons.map((icons)=>{
+    const iconsimages = newMultimedia('img',icons.icon.src,icons.icon.alt,['footer__icons']);
```

Comment:
please add a space between each parameter

Example:
`const iconsimages = newMultimedia('img', icons.icon.src, icons.icon.alt, ['footer__icons']);`

Reply:
Please Apply this suggestion to the entire file.

Reply:
Done. Applied and added the blank spaces between each parameter to the entire file.